django-rgd-imagery/client/rgd_imagery_client/plugin.py

```diff
@@ -101,11 +101,14 @@ def get_raster(self, raster_meta_id: Union[str, int, dict], stac: bool = False)
         if isinstance(raster_meta_id, dict):
             raster_meta_id = spatial_subentry_id(raster_meta_id)
 
+        r = self.session.get(f'rgd_imagery/raster/{raster_meta_id}').json()
         if stac:
-            r = self.session.get(f'rgd_imagery/raster/{raster_meta_id}/stac')
-        else:
-            r = self.session.get(f'rgd_imagery/raster/{raster_meta_id}')
-        return r.json()
+            # Get collection ID - TODO: if this is zero, will fail - could that happen?
+            collection_id = (
+                r['parent_raster']['image_set']['images'][0]['file']['collection'] or 'default'
+            )
```

Comment:
@mcovalt, should I check type here instead -- would a ID of 0 ever occur?

Reply:
It technically could occur, but unlikely since the ID sequence typically starts at 1 and monotonically increases.